server/src/metadata/object-metadata/hooks/before-create-one-object.hook.ts

```diff
@@ -1,11 +1,25 @@
-import { Injectable, UnauthorizedException } from '@nestjs/common';
+import {
+  ForbiddenException,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
 
 import {
   BeforeCreateOneHook,
   CreateOneInputType,
 } from '@ptc-org/nestjs-query-graphql';
 
 import { CreateObjectInput } from 'src/metadata/object-metadata/dtos/create-object.input';
+import { standardObjectsNames } from 'src/workspace/workspace-manager/standard-objects/standard-object-metadata';
+
+const OBJECT_TYPES = [
+  'featureflag',
```

Comment:
I think we should respect the case here:
featureFlag, refreshToken...

If a a user wants to name an object featureflag is it breaking the app?